ddtrace/internal/telemetry/metrics.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import abc
+import time
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import six
+from typing_extensions import Literal
+
+from ..hostname import get_hostname
+
+
+MetricType = Literal["count", "gauge", "rate"]
+MetricTagType = Dict[str, str]
+
+
+class Metric(six.with_metaclass(abc.ABCMeta)):
```

Comment:
Is there anything in https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/internal/metrics.py that we can reuse here?

Reply:
mmm, I don't think so, they have the same name but they're totally different